lms/product/product.py

```diff
@@ -58,12 +64,20 @@ class Product:
     # Accessor for external consumption
     Family = Family
 
+    api_client_name: str = field(init=False)
+    """Name of the client registered to access this LMS API"""
```

Comment:
Specifically this needs to be the name of a registered service that can be used with `request.find_service`.